src/F2_autoreport.py

```diff
@@ -500,8 +505,8 @@ def create_app(results_json):
     warnings_dict = {}
     errors_dict = {}
 
-    log_file = os.path.join(OUTPUT_FOLDER, "mvs_logfile.log")
-    # log_file = "/home/mr/Projects/mvs_eland/MVS_outputs/mvs_logfile.log"
+    # log_file = os.path.join(OUTPUT_FOLDER, "mvs_logfile.log")
+    log_file = "/home/mr/Projects/mvs_eland/MVS_outputs/mvs_logfile.log"
```

Comment:
You should not change an absolute path with variable with an absolute path to your local computer, at least not commit it